### DISCUSSION TITLE
How to use historic data from fusion solar as initial values?

### DISCUSSION THREAD

**Discussion by gitMatthias** [JensenNick/huawei_solar_pees#54] [Forum]
Hi @JensenNick,
I just found your awesome PEES package.

I have had my huawei system running for some time now and it has already fed a lot of energy into the power grid.
But if I have understood correctly, the calculations start at zero from the first restart after installation.

Is there a way to take into account the historical data that I can read from the fusion solar homepage?
And if so, what is the best way to do this?

**Comment by JensenNick** [comment]
@gitMatthias You're welcome. There is no option to import historical data. The data is based on only 3 sensors from the Huawei Solar integration. The only way I can think of to solve this is to build your own historical data in a spreadsheet and then import it to your HA database. That would require quite a bit of work.

**Reply by gitMatthias** [reply]
Ok, thanks for the feedback, I was almost expecting that answer.
I think, I'll try to rebuild your calculations in a spreadsheet - a project for the rainy days ahead
